s3migrate/verify.py

```diff
@@ -0,0 +1,98 @@
+import os
+import sys
+
+from shared import make_bucket_map, get_mogile_files_from_database
+import boto3
+import dbm.gnu
```

Comment:
I learned about gdbm today. Cool!

Reply:
They are useful if you need to store a few gigs of data and it is really too much to store in memory. I wouldn't use them for any long term storage though.